test/functional/apps/xpack/index.js

```diff
@@ -1,49 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-export default function ({ getService, getPageObjects }) {
-  const log = getService('log');
-  const PageObjects = getPageObjects(['monitoring', 'settings']);
-
-  describe('dismiss x-pack', function () {
```

Comment:
:information_source: This file didn't do anything for a long time, since the x-pack hint didn't use Notifier for a long time. Also given the way the functional test runner works I think this actually was run after all other tests and anyway didn't effect any test (since this itself isn't a test).